### ISSUE TITLE
[Source Request] Manganelo/Mangakakalot

### ISSUE THREAD

**Issue by rsmyst** [xgi/houdoku-extensions#52]
### Acknowledgements

- [X] Verify that this extension has not already been added (check the Extensions tab).
- [X] Check whether there is another ticket requesting this source.
- [X] Write a concise title for this ticket in the field above.

### Source name

Mangakakalot/Manganelo (point to clone databases)

### Source URL

https://manganato.com/ or alternatively https://mangakakalot.com/

### Source language(s)

English

### Additional information

This source has the most amount of manga i've seen compared to other sites. would be a great addition!

**Comment by ScruffiX** [comment]
I agree. Although most stuff can be found on Mangadex or Mangalife there are always some exceptions that I can only find on Mangakakalot.
